ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -154,16 +143,19 @@ import Purchases
             }
 
             if let signedDiscountTimestamp = signedDiscountTimestamp {
+                let storeProduct = StoreProduct(sk1Product: product)
                 if #available(iOS 12.2, macOS 10.14.4, tvOS 12.2, *) {
                     guard let discount = self.discountsByProductIdentifier[signedDiscountTimestamp] else {
                         completion(nil, productNotFoundError(description: "Couldn't find discount.", userCancelled: false))
                         return
                     }
-                    Purchases.shared.purchaseProduct(product, discount: discount, hybridCompletion)
+                    Purchases.shared.purchase(product: storeProduct,
```

Comment:
another place we might wanna update the internal var names

Reply:
I missed this one, I'll update in https://github.com/RevenueCat/purchases-hybrid-common/pull/131/